Missing dependency for compositor-proxy in Dockerfile

The dockerfile for compositor-proxy refused to build until I added libgl1-mesa-dev to the dependencies. The error it gave was:

```
CMake Error at /usr/share/cmake-3.24/Modules/FindPkgConfig.cmake:607 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.24/Modules/FindPkgConfig.cmake:829 (_pkg_check_modules_internal)
  CMakeLists.txt:22 (pkg_check_modules)

-- Configuring incomplete, errors occurred!
See also "/app/build/CMakeFiles/CMakeOutput.log".
error Command failed with exit code 1.
```

--- a/compositor-proxy/Dockerfile
+++ b/compositor-proxy/Dockerfile
@@ -13,6 +13,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     libgbm-dev \
     libdrm-dev \
     libegl-dev \
+    libopengl-dev \
     libglib2.0-dev \
     libgstreamer1.0-dev \
     libgstreamer-plugins-base1.0-dev \

